<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyPro</class>
 <widget class="QMainWindow" name="MyPro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyPro</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-image: url(:/images/2.jpeg);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>400</y>
      <width>331</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="timeDateShow">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dateShow">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="timeShow">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="setTimeDateButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>250</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>设置时间日期</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setDateButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>300</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>设置日期</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setTimeButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>350</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>设置时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="calendarShow">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>490</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>waiting for time</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>350</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>HH:mm:ss</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>300</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>250</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>yyyy/M/d HH:mm:ss</string>
    </property>
   </widget>
   <widget class="QLabel" name="realTimeShow">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="dateShow_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="timeShow_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="timeDateShow_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="getDateButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>获取日期</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getTimeButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>获取时间</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getDateTImeButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>获取时间日期</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>271</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>819</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
